conf/evolutions/default/252.sql

```diff
@@ -0,0 +1,129 @@
+
+ALTER TABLE gsv_data
+    ADD CONSTRAINT fk_gsv_data_gsv_link
+        FOREIGN KEY (gsv_panorama_id) REFERENCES gsv_link(gsv_panorama_id);
+
+ALTER TABLE label
+    ADD CONSTRAINT fk_label_gsv_data
+        FOREIGN KEY (gsv_panorama_id) REFERENCES gsv_data(gsv_panorama_id);
+
+ALTER TABLE label
+    ADD CONSTRAINT fk_label_street_edge
+        FOREIGN KEY (street_edge_id) REFERENCES street_edge(street_edge_id);
+
+ALTER TABLE label
+    ADD CONSTRAINT fk_label_user
+        FOREIGN KEY (user_id) REFERENCES sidewalk_user(user_id);
+
+ALTER TABLE label_point
+    ADD CONSTRAINT fk_label_point_label
+        FOREIGN KEY (label_id) REFERENCES label(label_id);
+
+ALTER TABLE region_completion
+    ADD CONSTRAINT fk_region_completion_region
+        FOREIGN KEY (region_id) REFERENCES region(region_id);
+
+ALTER TABLE street_edge_issue
+    ADD CONSTRAINT fk_street_edge_issue_street_edge
+        FOREIGN KEY (street_edge_id) REFERENCES street_edge(street_edge_id);
+
+ALTER TABLE street_edge_issue
+    ADD CONSTRAINT fk_street_edge_issue_user
+        FOREIGN KEY (user_id) REFERENCES sidewalk_user(user_id);
+
+ALTER TABLE street_edge_region
+    ADD CONSTRAINT fk_street_edge_region_region
+        FOREIGN KEY (region_id) REFERENCES region(region_id);
+
+ALTER TABLE street_edge_region
+    ADD CONSTRAINT fk_street_edge_region_street_edge
+        FOREIGN KEY (street_edge_id) REFERENCES street_edge(street_edge_id);
+
+ALTER TABLE user_current_region
+    ADD CONSTRAINT fk_user_current_region_region
+        FOREIGN KEY (region_id) REFERENCES region(region_id);
+
+ALTER TABLE user_current_region
+    ADD CONSTRAINT fk_user_current_region_user
+        FOREIGN KEY (user_id) REFERENCES sidewalk_user(user_id);
+
+ALTER TABLE user_login_info
+    ADD CONSTRAINT fk_user_login_info_login_info
+        FOREIGN KEY (login_info_id) REFERENCES login_info(login_info_id);
+
+ALTER TABLE user_password_info
+    ADD CONSTRAINT fk_user_password_info_login_info
+        FOREIGN KEY (login_info_id) REFERENCES login_info(login_info_id);
+
+ALTER TABLE user_role
+    ADD CONSTRAINT fk_user_role_role
+        FOREIGN KEY (role_id) REFERENCES role(role_id);
+
+ALTER TABLE user_role
+    ADD CONSTRAINT fk_user_role_user
+        FOREIGN KEY (user_id) REFERENCES sidewalk_user(user_id);
+
+ALTER TABLE user_survey_option_submission
+    ADD CONSTRAINT fk_user_survey_option_submission_survey_option
+        FOREIGN KEY (survey_option_id) REFERENCES survey_option(survey_option_id);
+
+ALTER TABLE validation_task_interaction
+    ADD CONSTRAINT fk_validation_task_interaction_gsv_data
+        FOREIGN KEY (gsv_panorama_id) REFERENCES gsv_data(gsv_panorama_id);
+
+ALTER TABLE validation_task_interaction
+    ADD CONSTRAINT fk_validation_task_interaction_mission
+        FOREIGN KEY (mission_id) REFERENCES mission(mission_id);
+
+# --- !Downs
+
+DROP CONSTRAINT IF EXISTS fk_validation_task_interaction_mission;
```

Comment:
Did the Downs work for you? I'm pretty sure that you need to specify the name of the table? So it would be:
```
ALTER TABLE validation_task_interaction DROP CONSTRAINT IF EXISTS fk_validation_task_interaction_mission;
```